.github/workflows/test.yml

```diff
@@ -146,3 +146,28 @@ jobs:
             nslookup \
             -- \
             nslookup kubernetes.default.svc.cluster.local
+test-macos-qemu-driver:
+    runs-on: macos-15
```

Comment:
macos-15 is M1 machine and it does not support nested virtualization. The only way to run vfkit is on macos-13 (intel).

Example from vmnet-helper integration test:
https://github.com/nirs/vmnet-helper/blob/main/.github/workflows/integration.yaml